src/picongpu/submit/taurus-tud/k80_profile.tpl

```diff
@@ -24,18 +24,18 @@ TBG_queue="gpu"
 TBG_mailSettings="ALL"
```

Comment:
hm, `"NONE"` should [actually work](https://www.dkrz.de/Nutzerportal-en/doku/mistral/running-jobs/slurm-introduction)